website/pages-parts/LandingPage/Testimonials.tsx

```diff
@@ -39,12 +47,17 @@ export function Testimonial({
             : styles['border-disable-top-right'],
         )}
       >
-        <h2>{who}</h2>
+        <img src={avatar} alt={who} className={styles['testimonial-avatar']} />
         <p>
-          <b>{company}</b>
+          <b>{who}</b>
+          <br />
+          {position} at <b>{company}</b>
           <br />
-          {position}
         </p>
+        <div className={styles['testimonial-links-wrapper']}>
+          <IconLink color="#412a94" icon={GitHub} to={linkGithub} />
+          <IconLink color="#412a94" icon={Linkedin} to={linkLinkedin} />
```

Comment:
I'd rather keep the color in the CSS. _(It shouldn't be a problem, as `react-feather` provides SVG icons with `currentColor` color by default, meaning that we can style them using the `color` property in CSS.)_

Reply:
My suggestion was the Badge component. 
In my opinion, we should keep the option (as an option) of giving an icon color using the props "color". In the current case I added our color --dusk-blue for testimonial icons using className. I know I could do this by specifying the color of the testimonial icons wrapper. However, it would not fit for me with the concept of reusability of the <IconLink /> component which is a separate organism with its properties. However, if I had to change anything here, write me.

Reply:
I simply don't see a need for that _now_, and there's no point in making the implementation more complex only because we _may_ need it. If we will, we'll do it in the same way - with CSS.

Reply:
You're right. Giving colors using CSS is more beneficial for us, because we have a predefined color palette there. I will remove the ability to color icons in IconLink using props color. leave the rest as is?

Reply:
Let's remove this `color` prop and move the `color` from `.testimonial-icon-color` to `.testimonial-links-wrapper` - it should work in the same way but with less code.

Reply:
Applying a color to a container (.testimonial-links-wrapper) doesn't work because the <a /> tag has default styling in the application. !important does not resolve this problem. I suggest leaving the props className passed in IconLink in testimonial (.testimonial-icon-color).

Reply:
Edit: i remove this className prop and add style by ```.testimonial-links-wrapper a {
  color: ...;
}```